python/pyspark/ml/tuning.py

```diff
@@ -288,6 +289,172 @@ def copy(self, extra=None):
         return CrossValidatorModel(self.bestModel.copy(extra))
 
 
+class TrainValidationSplit(Estimator, HasSeed):
+
+    estimator = Param(Params._dummy(), "estimator", "estimator to be tested")
+    estimatorParamMaps = Param(Params._dummy(), "estimatorParamMaps", "estimator param maps")
+    evaluator = Param(
+        Params._dummy(), "evaluator",
+        "evaluator used to select hyper-parameters that maximize the metric")
+    trainRatio = Param(Params._dummy(), "trainRatio", "proportion for train-validation ratio")
```

Comment:
The description for this parameter in scala is "ratio between training set and validation set (>= 0 && <= 1)". It would be good to maintain consistency here.